### PR TITLE
[ENH] Repository cleanup

### PR DESCRIPTION
**Pull request by robertcv**
Some cleanup before the next releas.
##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by robertcv** [comment]
From what I have read on the WWW the `package_data` and `include_package_data` are not needed if you have your data files declared in `MANIFEST.in`. If anybody can confirm this I would like to remove them.

**Comment by JakaKokosar** [comment]
regarding manifest: https://stackoverflow.com/a/57437660

File: setup.py [hunk 1]
```diff
@@ -44,10 +44,6 @@ def _discover_tests():
         long_description=LONG_DESCRIPTION,
         license='GPL-3.0',
         packages=find_packages(),
-        package_data={
-            'orangecontrib.geo.widgets': ['icons/*',
```
Comment (janezd) [inline setup.py:48]: I'm not an expert in setup files. Why can "icons/*" be removed from this list?

Comment (robertcv) [reply setup.py:48]: Because they are already in manifest.in.

**Pull request merged**